UnityProject/Assets/Scripts/UI/Lobby/GUI_LobbyDialogue.cs

```diff
@@ -67,6 +67,14 @@ void Start()
 			ShowLoginScreen();
 		}
 
+		private void Update() {
+			if ( Input.GetKeyDown( KeyCode.F6 ) )
```

Comment:
Requires a check to see if this is a Release build or a Development build. You can do this with `BuildPreferences.isForRelease`

Reply:
which is set to true on our build server